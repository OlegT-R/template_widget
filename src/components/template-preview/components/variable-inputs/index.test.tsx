import React from 'react';

import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { VariableInputs } from '.';

const onChangeVariable = () => null;
const arrVarNames = ['var1', 'var2'];

test('<VariableInputs />', () => {
  const component: any = shallow(
    <VariableInputs
      onChangeVariable={onChangeVariable}
      arrVarNames={arrVarNames}
    />
  );
  expect(toJson(component)).toMatchSnapshot();
});
