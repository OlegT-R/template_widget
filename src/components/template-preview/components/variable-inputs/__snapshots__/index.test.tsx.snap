// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VariableInputs /> 1`] = `
<styled.div>
  <styled.span>
    Variables:
  </styled.span>
  <Styled(Input)
    addonBefore="var1"
    key="var1"
    onChange={[Function]}
  />
  <Styled(Input)
    addonBefore="var2"
    key="var2"
    onChange={[Function]}
  />
</styled.div>
`;
